### ISSUE TITLE
Can't import mixins

### ISSUE THREAD

**Issue by s4msung** [pug-php/pug#121]
When including a template which declares a mixin, it's undefined in the current template.

I have packaged a demonstration:
[sample.zip](https://github.com/pug-php/pug/files/833954/sample.zip)

Just run:
```
php run.php
```
it should shout about undefined index *item_mixin_mixin*

**Comment by kylekatarnls** [comment]
Pug 2.5 handle only blocks after an extends, so you just have to put your include in each block that use the mixin:

```pug
extends main

block content
    include item_mixin

    .container
        h2 Example pages
        ul.navlist
            each item in items
                +item_mixin(item)
```

 It will be fixed in 3.0.

**Comment by kylekatarnls** [comment]
You can update to `3.0.0-alpha2`, it should be fixed. Please don't hesitate to test the new pug-php 3 and give us your feedback.
